content/en/continuous_testing/environments/multiple_env.md

```diff
@@ -0,0 +1,118 @@
+  tag: "Blog"
+  text: "Test internal applications with Datadog's testing tunnel and private locations"
+- link: "/continuous_testing/environments/proxy_firewall_vpn"
+  tag: "Documentation"
+  text: "Learn about testing while using proxies, firewalls, or VPNs"
+---
+
+## Overview
+
+Continuous Testing allows you to apply the same scenario from scheduled tests against the production environment to development and staging environments. Continuous Testing uses Synthetic tests throughout the development cycle to ensure regressions are caught as soon as possible.
+
+When triggering a CI test, you can overwrite the starting URL of a [browser][1] or [API test][2] to reroute the Synthetic Worker to the appropriate environment.
+
+## Overriding the starting URL
+
+A Synthetic browser test starts the test scenario by navigating to a starting URL. Similarly, an API HTTP test sends a request to a specific URL. When triggering a CI test, you can overwrite this starting URL to point to another environment where your application is deployed in.
+
+{{< img src="continuous_testing/continuous_testing_start-url_substitution.png" alt="Continuous Testing tunnel allows the Synthetics Worker to reach your private applications" width="100%" >}}
+
+When triggering a CI test, the `startUrl` field allows you to overwrite the first URL that a browser test navigates to or the URL used by an HTTP test request. You can specify this option through the global configuration file, the Synthetics configuration files (`*.synthetics.json`), or the command line flag `--start-url`.
+
+```shell
+yarn datadog-ci synthetics run-tests --public-id <public-id> --start-url "https://staging.my-app.com"
+```
+
+
+This option allows you to reuse the same test scenario on both the production environment and other development environments (such as staging) as long as they are publicly available. To learn how to test against [private environments][4], see [Testing While Using Proxies, Firewalls, or VPNs][3].
+
+## Partially modifying the starting URL
+
+If some of your tests start at the homepage, or a similarly simple URL, the previous solution works fine, but it doesn't cover every use case. Blindly replacing the starting URL may unintentionally remove the path or certain search query parameters from the URL that the scenario is expected to test.
+
+In addition to `startUrl`, the `startUrlSubstitutionRegex` field allows you to modify the starting URL without overwriting it entirely. This option allows you to substitute parts of the default starting URL based on the provided regular expression.
+
+This field expects a string containing two parts, separated by a pipe character `|`: `<regex>|<rewriting rule>`. The first part is the regex to apply to the default starting URL. The second is the expression to rewrite the URL.
+
+A simple example looks like the following:
+
+```shell
+https://prod.my-app.com/(.*)|https://staging.my-app.com/$1
+```
```

Comment:
Can you remove the `shell` annotation? Apparently the documentation preprocessor adds a visual `$` at the beginning to make people understand it's  a command line. But here it's misleading:

![image](https://github.com/DataDog/documentation/assets/9317502/19ec4b86-1fdc-4141-b1d7-2136f4736c39)